src/main/java/org/terasology/structureTemplates/internal/ui/StructureTemplateEditorScreen.java

```diff
@@ -165,21 +167,25 @@ private void updateBlockEditingProcessOnServer() {
     }
 
     private void onEditTemplatePropertiesButton(UIWidget button) {
-        StructureTemplatePropertiesScreen screen = getManager().pushScreen("StructureTemplates:StructureTemplatePropertiesScreen", StructureTemplatePropertiesScreen.class);
+        StructureTemplatePropertiesScreen screen = getManager().pushScreen("StructureTemplates" +
+                ":StructureTemplatePropertiesScreen", StructureTemplatePropertiesScreen.class);
         screen.setEditorEntity(getInteractionTarget());
     }
 
 
     private void onMakeBoxShapedButton(UIWidget button) {
         EntityRef entity = getInteractionTarget();
         StructureTemplateOriginComponent component = entity.getComponent(StructureTemplateOriginComponent.class);
+        //TODO: The region might be invalid (empty) - what should we do in this case?
```

Comment:
This was also the case before, just wanted to note this down just in case this becomes a problem some time.